addons/addon-base-raas/packages/base-raas-services/lib/aws-accounts/aws-cfn-service.js

```diff
@@ -77,12 +76,13 @@ class AwsCfnService extends Service {
 
     const permissionsTemplateRaw = await cfnApi.getTemplate(params).promise();
 
-    return {
-      permissionsTemplateStr: permissionsTemplateRaw.TemplateBody,
-    };
+    return permissionsTemplateRaw.TemplateBody;
   }
 
-  async checkAccountPermissions(requestContext, accountEntity) {
+  async checkAccountPermissions(requestContext, accountId) {
+    const awsAccountsService = await this.service('awsAccountsService');
```

Comment:
Consider moving the assertAuthorized call to the beginning of checkAccountPermissions and batchCheckAccountPermissions methods? Adding authorization to the service method call from the router makes it clearer to reader where/how authorization is done.

Reply:
Sure, that makes sense!